### ISSUE TITLE
duplicated help_block on entity type collection form

### ISSUE THREAD

**Issue by SamiErrighi** [kristijanhusak/laravel-form-builder#177]
Hi,

I'am working with collection child form and when I use the entity type i have a duplicated help_block.

```
<?php
use Kris\LaravelFormBuilder\Form;

class TagsForm extends Form
{
    public function buildForm()
    {
        $this
            ->add('name', 'text')
            ->add('page_id', 'entity', [
                'class' => 'App\Entity\Page',
                'property' => 'title',
             ])
            ->add('desc', 'textarea');
    }
}

class PostForm extends Form
{
    public function buildForm()
    {
        $this
            ->add('title', 'text')
            ->add('body', 'textarea')
            ->add('tags', 'collection', [
                'type' => 'form',
                'options' => [   
                    'class' => 'App\Forms\TagsForm'
                    'label' => false,
                ]
            ]);
    }
}

```
I work on 1.6.30 branch

**Comment by kristijanhusak** [comment]
Hi,

Thanks for reporting. I'll check it out. Just a few questions:

* Did you use any previous version before?
* If you did, have you published views?

**Comment by SamiErrighi** [comment]
Hi,
Yes I published my views when I updated the vendor

**Comment by kristijanhusak** [comment]
Did you use any older versions or you started using package from 1.6.30?

**Comment by SamiErrighi** [comment]
I started from 1.6.10

**Comment by kristijanhusak** [comment]
Ok, you need to update your views, that bug was fixed in 1.6.30 (changelog [here](https://github.com/kristijanhusak/laravel-form-builder/blob/master/CHANGELOG.md#1630)), but it requires views update.
If you didn't change anything in published views, just delete them and republish if you need.

If you have changed something, update them according to views here https://github.com/kristijanhusak/laravel-form-builder/tree/master/src/views

**Comment by SamiErrighi** [comment]
Ok thanks that's work now .
